releasing/release_repositories.md

```diff
@@ -0,0 +1,8 @@
+# Release repositories
+
+TODO: create the real document with information about:
```

Comment:
Does this need to be addressed? One of the links in `release_draft.md` leads to this file